MAUI/NavigationDrawer/Overview.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: About .NET MAUI Navigation Drawer control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control, its elements and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+
+# .NET MAUI Navigation Drawer (SfNavigationDrawer) Overview
+
+The NavigationDrawer for .NET MAUI is a simpler component to create a navigation pane in the application. It consists of a content area and a sliding pane that slides out from the edge of the page. The pane can be opened by swiping the edges of the screen or programmatically.
+
+## Key Features
+
+* Pane positions – Supports pane position in all four directions such as Left, Right, Top and Bottom. 
```

Comment:
* Pane positions – Supports pane position in all four directions: Left, Right, Top, and Bottom.

Reply:
Changed